runtime/doc/remote_plugin.txt

```diff
@@ -101,35 +101,34 @@ the remote plugin manifest must be generated with `:UpdateRemotePlugins`.
 4. Remote plugin manifest			    *remote-plugin-manifest*
 
 Just installing remote plugins to "rplugin/{host}" isn't enough for them to be
-automatically loaded when required. The `:UpdateRemotePlugins` command must be
-executed every time a remote plugin is installed, updated, or deleted.
+automatically loaded when required. You must execute `:UpdateRemotePlugins`
+every time a remote plugin is installed, updated, or deleted.
```

Comment:
I like how my suggestion assumes you know what to do with the command, as opposed to explicitly saying `You must execute ...`.

Anyways, I don't want to block this PR with what's arguably pretty subjective, so please feel free to leave it as is.